.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*.scss": "npm run lint:scss",
+  "*.md": "npm run lint:md",
+  "*.js": "npm run lint:js"
```

Comment:
Haven't tested it yet, but from looking at the commands I think this is not yet only linting files that changes but if one js file changed for example still the full JS codebase would be linted.

Reply:
It actually does magically work correctly! lint-staged automatically passes each filename as a parameter to the commands above. :-)